src/lib/useOrientationChange.js

```diff
@@ -0,0 +1,22 @@
+import { useEffect } from 'react'
+import { useStore } from '../store'
+
+export function useOrientationChange() {
+  const set = useStore((state) => state.set)
+  const isMobilePortrait = useStore((state) => state.isMobilePortrait)
+  useEffect(() => {
+    const handler = () => {
+      if (window.orientation === 0) {
```

Comment:
orientation property is also deprecated. I think within the `src/lib.isPotraitMode.js` you can set the` matchMedia('(orientation: portrait)')` to a new `const potraitMedia = matchMedia('(orientation: portrait)')` and then use `potraitMedia.matches`.

Something like:
```javascript
 const handler = () => set({isMobilePortrait: potraitMedia.matches}))
 ```
 
 should do the job.